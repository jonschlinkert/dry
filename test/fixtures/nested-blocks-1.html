{% extends "nested-blocks-1" %}

{% block "a" %}
a: "nested-blocks-1.html"
{% endblock %}

{% block "a.b" %}
a.b: "nested-blocks-1.html"
{% endblock %}