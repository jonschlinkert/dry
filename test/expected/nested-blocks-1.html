
a: "nested-blocks-1.html"
